tool/run.py

```diff
@@ -12,6 +12,8 @@
 from tool.model import Result, Submission
 from tool.utils import BColor
 
+_MAX_OUTPUT_LEN = 20
```

Comment:
I'd prefer having a cli argument instead of a hard coded value. I personally use the tool to solve the problems and submit my answer, and I know some folks do the same, it wouldn't be a good idea to truncate answers in that case

Reply:
you're right. This was more of a solution to the day 8 part 2 problem where the output was a series of digits that do not form an answer if taken as is. Given the work that Jon did to resolve that issue (parser), this PR doesn't do much anymore. If you still think this feature could come in handy, I could add a cli argument (since I agree with what you said).

Reply:
I'll close it for now, feel free to reopen it if the feature seems interesting